PROVESFlightControllerReference/test/int/common.py

```
@@ -40,6 +40,26 @@ def set_default_retries(n: int) -> None:
+    sensor read on that face then fails with EXECUTION_ERROR.  Sending
+    EXIT_SAFE_MODE first clears the condition long enough for a short test to
+    run (auto-entry re-arms only after SafeModeDebounceSeconds).
+
+    Best-effort: EXIT_SAFE_MODE is a no-op when the board is already in NORMAL,
+    and a failure here should surface as the real test's failure, not as a
+    setup error.
+    """
+    try:
+        fprime_test_api.send_command("ReferenceDeployment.modeManager.EXIT_SAFE_MODE")
+    except Exception:  # noqa: BLE001 - advisory only; the test itself is the assertion
+        pass
```

Ruff (0.16.0)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)